config/locales/en.yml

```diff
@@ -3754,14 +3755,14 @@ en:
           verification_code_label: Enter the 6-digit code
       w2:
         edit:
-          box14_explanation:
-            md_html: For tax year %{year}, we will only process the <strong>STPICKUP</strong> code, which represents any contributions you made to the state retirement system.
+          box14_explanation_md_html: For tax year %{year}, we will only process the <strong>STPICKUP</strong> code, which represents any contributions you made to the state retirement system.
```

Comment:
Typo toward the end of the text - "those don''t need to be reported"

Reply:
It looks wrong, but the double apostrophe `''` is to escape the `'` since the whole string is wrapped in single-quotes `'`. This was actually done automatically by the i18n-normalize script, and you can see in one of the screenshots that it renders as a single apostrophe.